src/engine/platform/opl4.h

```diff
@@ -0,0 +1,115 @@
+/**
```

Comment:
Will get rid of this soon. I plan to do OPL4 via OPL3+MultiPCM.

Reply:
Will be addressed later.